### PR TITLE
[Merged by Bors] - chore: uncomment some now viable assert_not_exists

### PR DESCRIPTION
**Pull request by kim-em**
Per [zulip](https://leanprover.zulipchat.com/#narrow/stream/345428-mathlib-reviewers/topic/.60assert_not_exists.60.20at.20top.20or.20bottom.20of.20file.3F/near/459960523) discussion.

### CODE REVIEW

File: Mathlib/Algebra/GroupWithZero/Conj.lean [hunk 1]
```diff
@@ -11,7 +11,6 @@ import Mathlib.Algebra.GroupWithZero.Units.Basic
 -/
 
 assert_not_exists Multiset
--- TODO
```
Comment (grunweg) [inline Mathlib/Algebra/GroupWithZero/Conj.lean:14]: Why this change?

**Review by grunweg — COMMENTED** [review]
All changes so far look good to me (with one exception); the build fails, though.

File: Mathlib/Algebra/GroupWithZero/Conj.lean [hunk 2]
```diff
@@ -11,7 +11,6 @@ import Mathlib.Algebra.GroupWithZero.Units.Basic
...
 -- assert_not_exists DenselyOrdered
```
Comment (jcommelin) [inline Mathlib/Algebra/GroupWithZero/Conj.lean:14]:
```suggestion
-- TODO
-- assert_not_exists DenselyOrdered
```

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge
